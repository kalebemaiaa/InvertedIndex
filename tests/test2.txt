QueryAnd(QueryWord("senador"), QueryOr(QueryWord("governador"), QueryWord("prefeito")))
1816.raw
5545.raw
4596.raw
2360.raw
11498.raw
2015.raw
3060.raw
1185.raw
320.raw
1021.raw
954.raw
5198.raw
6342.raw
518.raw
1791.raw
3548.raw
5496.raw
12005.raw
360.raw
82.raw
1146.raw
6303.raw
4480.raw
2051.raw
6090.raw
4581.raw
4358.raw
131.raw
11810.raw
2610.raw
550.raw
11875.raw
3838.raw
952.raw
562.raw
8740.raw
3771.raw
866.raw
1001.raw
4282.raw
2241.raw
3459.raw
10811.raw
1150.raw
8794.raw
2529.raw
6364.raw
6016.raw
5833.raw
2119.raw
1612.raw
11098.raw
1624.raw
6359.raw
4044.raw
903.raw
11251.raw
11303.raw
527.raw
344.raw
5900.raw
11180.raw
5486.raw
3789.raw
5524.raw
11602.raw
11621.raw
4736.raw
3360.raw
902.raw
3915.raw
11901.raw
1803.raw
3751.raw
6317.raw
4340.raw
2975.raw
1407.raw
8792.raw
3837.raw
5447.raw
5255.raw
496.raw
1327.raw
513.raw
215.raw
2797.raw
4825.raw
4663.raw
2607.raw
3085.raw
731.raw
3675.raw
3184.raw
11568.raw
1676.raw
1502.raw
11142.raw
2345.raw
10823.raw
4203.raw
3946.raw
1543.raw
4204.raw
1470.raw
1050.raw
11194.raw
5091.raw
4522.raw
11104.raw
1534.raw
11168.raw
774.raw
10902.raw
6223.raw
2288.raw
11193.raw
365.raw
1601.raw
1903.raw
4805.raw
3435.raw
1426.raw
11653.raw
5206.raw
8840.raw
6391.raw
4073.raw
4111.raw
3341.raw
164.raw
5666.raw
3343.raw
5839.raw
2856.raw
1247.raw
11369.raw
11390.raw
4730.raw
3033.raw
2465.raw
2978.raw
2886.raw
33.raw
6254.raw
5249.raw
1107.raw
4838.raw
4690.raw
4636.raw
11400.raw
11117.raw
2823.raw
5242.raw
2354.raw
5788.raw
6238.raw
11481.raw
1681.raw
1256.raw
1246.raw
3815.raw
3163.raw
11715.raw
2966.raw
12042.raw
3267.raw
1690.raw
6332.raw
8747.raw
11226.raw
3122.raw
5842.raw
6236.raw
6197.raw
392.raw
11398.raw
11043.raw
4680.raw
3847.raw
6048.raw
10777.raw
11361.raw
8772.raw
5433.raw
6142.raw
159.raw
11911.raw
2851.raw
3437.raw
3405.raw
1422.raw
6260.raw
2811.raw
751.raw
11102.raw
11899.raw
2649.raw
1013.raw
2733.raw
