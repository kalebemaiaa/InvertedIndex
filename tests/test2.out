25
38
422
436
1152
1390
1400
1463
1701
2012
2178
2645
2656
2669
3167
3199
3239
3882
4043
4557
4748
4912
5047
5253
5826
5835
6231
6295
10854
11701
